.github/workflows/integer_full_benchmark.yml

```diff
@@ -35,7 +35,10 @@ jobs:
       max-parallel: 1
       matrix:
         command: [ integer, integer_multi_bit]
-        op_flavor: [ default, default_comp, default_scalar, default_scalar_comp, smart, smart_comp, smart_scalar, smart_parallelized, smart_parallelized_comp, smart_scalar_parallelized, unchecked, unchecked_comp, unchecked_scalar, unchecked_scalar_comp, misc ]
+        op_flavor: [ default, default_comp, default_scalar, default_scalar_comp,
+                     smart, smart_comp, smart_scalar, smart_parallelized, smart_parallelized_comp, smart_scalar_parallelized, smart_scalar_parallelized_comp,
+                     unchecked, unchecked_comp, unchecked_scalar, unchecked_scalar_comp,
```

Comment:
question between default (which are all parallelized at the moment) and unchecked, some of which parallelized what should be done for the naming here ?

Reply:
I see three solutions to this issue:

1. create a new group containing only `unchecked_scalar_[add|sub|mul]` operation and rename the other by adding the `parallelized` in the name.
2. write a parallel implementation for the operations mentioned above
3. remove from the benchmarks these 3 sequential operations.

Reply:
The way negation and subtraction are written is not parallelizable i think

we could do subtraction in a different way to have it parallelizable, however I don't really thing there any benefit to it (much like when testing, there does not seem to have any real benefit of using unchecked_add or unchecked_add_parallelized in the default_add_parallelized)